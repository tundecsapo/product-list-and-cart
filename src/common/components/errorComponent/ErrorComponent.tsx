import { CustomContainer } from "../customContainer/CustomContainer";
import Typography from "@mui/material/Typography";

export const ErrorComponent = () => {
  return (
    <CustomContainer>
      <Typography variant="h5" sx={{ mt: 3 }}>
        There was a problem while listing the products. Please, try again later.
      </Typography>
    </CustomContainer>
  );
};
