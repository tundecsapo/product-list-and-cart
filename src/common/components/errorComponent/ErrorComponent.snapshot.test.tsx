import renderer from "react-test-renderer";
import { ErrorComponent } from "./ErrorComponent";

it("renders correctly", () => {
  const tree = renderer.create(<ErrorComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
