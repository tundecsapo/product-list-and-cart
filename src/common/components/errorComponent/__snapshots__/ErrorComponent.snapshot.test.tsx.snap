// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
[
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-4a04zc-MuiContainer-root{width:100%;margin-left:auto;box-sizing:border-box;margin-right:auto;display:block;padding-left:16px;padding-right:16px;padding:32px;padding-top:96px;height:100vh;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;}@media (min-width:600px){.css-4a04zc-MuiContainer-root{padding-left:24px;padding-right:24px;}}@media (min-width:1536px){.css-4a04zc-MuiContainer-root{max-width:1536px;}}",
      }
    }
    data-emotion="css 4a04zc-MuiContainer-root"
  />,
  <div
    className="MuiContainer-root MuiContainer-maxWidthXl css-4a04zc-MuiContainer-root"
  >
    <style
      dangerouslySetInnerHTML={
        {
          "__html": ".css-xr009-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:400;font-size:1.5rem;line-height:1.334;letter-spacing:0em;margin-top:24px;}",
        }
      }
      data-emotion="css xr009-MuiTypography-root"
    />
    <h5
      className="MuiTypography-root MuiTypography-h5 css-xr009-MuiTypography-root"
    >
      There was a problem while listing the products. Please, try again later.
    </h5>
  </div>,
]
`;
